eZ/Publish/API/Repository/Tests/UserServiceTest.php

```diff
@@ -3045,6 +3022,21 @@ public function testExpireUserToken($userToken)
         $userService->loadUserByToken($userToken);
     }
 
+    /**
+     * Test trying to load User by invalid Token.
+     *
+     * @covers \eZ\Publish\API\Repository\UserService::loadUserByToken
+     */
+    public function testLoadUserByTokenThrowsNotFoundException()
+    {
+        $this->expectException(NotFoundException::class);
+
+        $repository = $this->getRepository();
+        $userService = $repository->getUserService();
+
+        $userService->loadUserByToken('not_existing_token');
```

Comment:
`expectExpection` statement should be placed right before the statement which is expected to trigger that exception